plugins/linear-genome-view/src/LinearGenomeView/components/LinearGenomeView.tsx

```diff
@@ -75,19 +76,8 @@ const LinearGenomeView = observer(({ model }: { model: LGV }) => {
           handleClose={() => model.setSearchResults(undefined, undefined)}
         />
       ) : null}
-      {!hideHeader ? (
```

Comment:
is there a reason for the mllv why hideHeader got moved into the minicontrols/header?

Reply:
Yes, MLLV has its own conditions for rendering the header, and if I recall correctly moving these conditions into their own components fixed an issue where the header was being duplicated or not shown at all.